components/client/src/test/unit/java/com/hotels/styx/client/StyxHttpClientTest.java

```diff
@@ -90,9 +90,10 @@ public void tearDown() {
      */
     @Test
     public void cannotBeModifiedAfterCreated() throws ExecutionException, InterruptedException {
-        StyxHttpClient client = new StyxHttpClient.Builder()
-                .userAgent("v1")
-                .build();
+        StyxHttpClient.Builder builder = new StyxHttpClient.Builder().userAgent("v1");
+        StyxHttpClient client = builder.build();
+
+        builder.userAgent("v2");
```

Comment:
This looks incorrect. I Looks like this change can be reverted back?

Reply:
that change was reverted yesterday